uaclient/status.py

```diff
@@ -518,7 +518,7 @@ def simulate_status(
     inapplicable_resources = [
         resource["name"]
         for resource in sorted(resources, key=lambda x: x["name"])
-        if not resource["available"]
+        if not resource["available"] or resource["available"] == "no"
```

Comment:
same change needs to be done for attached status maybe, on line `235` in this same file